local-volume/README.md

```diff
@@ -200,6 +200,21 @@ go run hack/e2e.go -- -v --test --test_args="--ginkgo.focus=\[Feature:LocalPersi
 * Avoid recreating nodes with the same node name while there are still old PVs
   with that node's affinity specified. Otherwise, the system could think that
   the new node contains the old PVs.
+* While leveraging the local PV provisioner, race conditions can occur when
```

Comment:
@msau42 - at the bottom of this bullet, I added a sentence and link to the related Kubernetes issue. Does this look good to merge now?

Reply:
Thinking about this some more, I don't think there is a workaround for this issue without a code change.  Even bind mounting requires that you create the destination directory beforehand.  So we must have a code change to only discover mounted directories.

Reply:
How about I remove the statements about race conditions, since we don't have a solution yet.

This change would leave our recommendations saying that bindmount should be used for mount points and symlinks should be used for block devices.

Reply:
I think we can just leave out the whole statement.  The final solution is looking like: bind mount for shared directories, and regular mount for mount points.  I don't think symlink is necessary for block.

Reply:
I removed the whole statement, including recommendations on bind mount vs regular mount.